include/zephyr/net/wifi.h

```diff
@@ -465,6 +467,24 @@ enum wifi_ps_wakeup_mode {
 /** Helper function to get user-friendly ps wakeup mode name. */
 const char *wifi_ps_wakeup_mode_txt(enum wifi_ps_wakeup_mode ps_wakeup_mode);
 
+/**
+ * @brief Wi-Fi power save exit strategy
+ */
+enum wifi_ps_exit_strategy {
+	/** PS-Poll frame based */
+	WIFI_PS_EXIT_CUSTOM_ALGO = 0,
+	/** QoS NULL frame based */
+	WIFI_PS_EXIT_EVERY_TIM,
+
+	/** Last value */
+	WIFI_PS_EXIT_LAST,
+	/** Maximum value */
+	WIFI_PS_EXIT_MAX = WIFI_PS_EXIT_LAST - 1,
```

Comment:
No need to do add these to documentation, we should surround them with `@cond ... @endcond`